challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |eachstr| eachstr.capitalize }
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.select { |dogstr| dogstr == "dog"  }
```

Comment:
`animal`, or `animal_str`, might be a better variable name here, since what you're checking is whether a given animal is the dog.